src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int arrayLength = 6;
+        Figure[] figures = new Figure[arrayLength];
+
+        for (int i = 0; i < 6; i++) {
+            figures[i] = (i < arrayLength / 2)
+                    ? figureSupplier.getRandomFigure()
+                    : figureSupplier.getDefaultFigure();
+        }
+
+        for (Figure figure : figures) {
+            figure.draw();
+        }
```

Comment:
Do we actually need a separate loop? Could it affect the performance, if we had an array of billions of elements?

Let's print figures in the previous loop after their creation.

Reply:
done